.changeset/hand-picked-string.md

```diff
@@ -1,4 +1,9 @@
 ---
+# This metadata needs to be removed before running generate:changelog
```

Comment:
you can put comments in here? Of since this is markdown, is that a title?

Reply:
the "front matter" is actually YAML, so it's a comment